lib/msal-common/src/request/CommonRefreshTokenRequest.ts

```diff
@@ -22,4 +23,6 @@ export type CommonRefreshTokenRequest = BaseAuthRequest & {
     refreshToken: string;
     ccsCredential?: CcsCredential;
     forceCache?: boolean;
+    tokenBodyParameters?: StringDict;
```

Comment:
I am also toying around the idea to add this in `BaseAuthRequest` as this is supposed to be a generic param to be added to the `body` of a request sent to `/token` end point. Do we have a clear demarcation of `tokenQueryParameters` vs `tokenBodyParameters`? 

These changes fix the bugs, however would like to consolidate the param addition at *many* places if `tokenBodyParameters` can be repurposed.